deploy/helm-chart/kubernetes-replicator/Chart.yaml

```diff
@@ -4,6 +4,6 @@ description: Controller for replicating secrets+configmaps across namespaces
 
 type: application
 
-version: 2.7.3
+version: 2.7.4
 
-appVersion: v2.7.3
+appVersion: v2.7.4
```

Comment:
Bumping the app/chart version manually is not necessary; this will be done by our release scripts when a new version is actually released.